modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitySetsMVCRenderCommand.java

```diff
@@ -63,6 +68,51 @@ public String render(
 		return "/manage_user_associated_data_entity_sets.jsp";
 	}
 
+	private List<UADEntitySetComposite> _getUADEntitySetComposites(
```

Comment:
Since the `UADEntitySetComposite` class is strictly for display logic and not an API to be implemented by the other `uad` modules, we should move it from the `api` module to the `web` module.